src/audio-buffer-mock.ts

```diff
@@ -7,10 +7,16 @@ export class AudioBufferMock implements IAudioBuffer {
 
     public sampleRate: number;
 
+    private channelData: Float32Array[];
+
     constructor({ length, numberOfChannels = 1, sampleRate }: IAudioBufferOptions) {
         this.length = length;
         this.numberOfChannels = numberOfChannels;
         this.sampleRate = sampleRate;
+        this.channelData = [];
+        for (let i = 0; i < this.numberOfChannels; i++) {
```

Comment:
What do you think about using a `Map` to store the `channelData` internally. Maybe with a signature like this:

```ts
private channelData: Map<number, Float32Array>;
```

This would allow us to not initialize a `Float32Array` unless needed. `getChannelData()` would then look like this:

```ts
getChannelData(channel: number): Float32Array {
    let channelData = this.channelData.get(channel);

    if (channelData === undefined) {
        channelData = new Float32Array(this.length);

        this.channelData.set(channel, channelData);
    }

    return channelData;
}
```